lua/weapons/weapon_lfsmissilelauncher.lua

```diff
@@ -199,7 +206,7 @@ function SWEP:PrimaryAttack()
 	self:SetNextPrimaryFire( CurTime() + 0.5 )
 	self:TakePrimaryAmmo( 1 )
 
-	timer.Simple( 0, function()
+	timer.Simple( 0.05, function()

```

Comment:
Nonzero timers are usually a code smell. Is this the best way to handle it?

Reply:
Changed it, The reload will happen when you try to fire with an empty clip instead.